### PR TITLE
separate nodes.js file into separate files #9 + add config.js

### PR DESCRIPTION
**Pull request by LucasMatuszewski**
As we discussed in #6 - I reimplement that PR without TS (but I hope we will add TS in the next PR ;)

---

Add new node classes and register them in nodes.js

* **LLMNode**: Define and export the `LLMNode` class, import colors from `configs.js`, and use imported colors.
* **TextToSpeechNode**: Define and export the `TextToSpeechNode` class, import colors from `configs.js`, and use imported colors.
* **AudioInputNode**: Define and export the `AudioInputNode` class, import colors from `configs.js`, and use imported colors.
* **TextInputNode**: Define and export the `TextInputNode` class, import colors from `configs.js`, and use imported colors.
* **AudioOutputNode**: Define and export the `AudioOutputNode` class, import colors from `configs.js`, and use imported colors.
* **STTNode**: Define and export the `STTNode` class, import colors from `configs.js`, and use imported colors.
* **TTSNode**: Define and export the `TTSNode` class, import colors from `configs.js`, and use imported colors.
* **nodes.js**: Import and register all new node classes.
* **configs.js**: Move `languageOptions` and export an object to store HSL colors with keys as color names.

I propose HSL colors because they are much more readable than RGB or Hex. You can easily visualize the first number as a 360-degree hue circle of colors, the second number is saturation of the color in 0-100%, and the last one is lightness 0-100%. It's very easy to generate a nice palate of color just by changing the light value to get similar colors but in light/dark versions. TailwindCSS popularised HSL recently and IMHO it's a good new standard.

### CODE REVIEW

File: src/nodes/TextToSpeechNode.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        this.title = "Text to Speech";
+        this.addInput("text", "string");
+        this.addOutput("audio", "audio");
+        this.color = colors.yellowDark;
+    }
+
+    onExecute() {
+        const text = this.getInputData(0);
+        // Implement TTS logic here
+        this.setOutputData(0, { type: "audio", data: "synthesized_audio_data" });
```
Comment (LucasMatuszewski) [inline src/nodes/TextToSpeechNode.js]: We probably should just delete this file, it duplicates TTSNode.

I guess it was only for temporary test purposes? Or rename it as `TTSNodeTest.js` to make it clear that this is test node?

**Comment by LucasMatuszewski** [comment]
UP :)
@vipyne @hyypeman

File: src/nodes/TextToSpeechNode.js (hunk 1 above)
Comment (vipyne) [reply src/nodes/TextToSpeechNode.js]: +1 to delete it.  I believe it was simply a consequence of moving fast at a hackathon.

**Comment by hyypeman** [comment]
LGTM

File: src/nodes/TextToSpeechNode.js (hunk 1 above)
Comment (LucasMatuszewski) [reply src/nodes/TextToSpeechNode.js]: Ok @vipyne - I deleted TextToSpeechNode - we can merge this PR if you don't have any other suggestions.

And pls let me know what are the future plans - how can I help :)

**Pull request merged**